pages/linux/bmon.md

```diff
@@ -0,0 +1,19 @@
+# bmon
+
+> Monitor bandwidth and capture network related statistics.
+
+- Display the list of all the interfaces:
+
+`bmon -a`
+
+- Display rates in bits per second:
+
+`bmon -b`
+
+- Set policy to define which network interface(s) is/are displayed:
+
+`bmon -p {{interface_1,interface_2,interface_3}}`
+
+- Set interval (in seconds) in which rate per counter is calculated:
```

Comment:
2.5 is a bit odd. Can we make it 2 ?

Reply:
Updated the value to be 2.0 instead of 2, to indicate that floating point values can be passed as well.